content/docs/get_started/login_requiring.md

```diff
@@ -0,0 +1,166 @@
+          <button on="tap:amp-access.login-sign-in" class="button-primary comment-button">Login</button>
+</span> 
+[/sourcecode]
+
+The behaviour of `amp-access` related attributes are dependent on a page-wide configuration for `amp-access`, in our case, this one:
+
+[sourcecode:html]
+<script id="amp-access" type="application/json">
+  {
+    "authorization": "https://ampbyexample.com/samples_templates/comment_section/authorization?rid=READER_ID&url=CANONICAL_URL&ref=DOCUMENT_REFERRER&_=RANDOM",
+    "noPingback": "true",
+    "login": {
+      "sign-in": "https://ampbyexample.com/samples_templates/comment_section/login?rid=READER_ID&url=CANONICAL_URL",
+      "sign-out": "https://ampbyexample.com/samples_templates/comment_section/logout"
+    },
+    "authorizationFallbackResponse": {
+      "error": true,
+      "loggedIn": false
+    }
+  }
+</script>
+[/sourcecode]
+
+The authorization endpoint is deployed as part of AMPByExample. It’s in fact responsibility of the publisher of the page to provide this endpoint. In the sample case, for simplicity sake, we implemented a simple logic where when this request is received, the server reads the value of a cookie called `ABE_LOGGED_IN`. If the cookie is not there, we return a JSON response containing `loggedIn = false`. As result, the first time a user lands on the page, this request will return `loggedIn = false` and the login button will be shown.
+
+Looking again at the button's HTML code, by using `on="tap:amp-access.login-sign-in"`, we specify that once the user taps on the button, the URL specified in the JSON above should be used:
+
+[sourcecode:json]
+{ 
+	"login": {
+          "sign-in": "https://ampbyexample.com/samples_templates/comment_section/login?rid=READER_ID&url=CANONICAL_URL"
+  }
+}
+
+[/sourcecode]
+
+{% call callout('Note', type='success') %}
+Notice that it’s possible to define different URLs inside the login node, in this case we are defining `sign-in`, we will later define `sign-out`
+{% endcall %}
+
+The login page is a non-AMP page in which we populate the login and password values for the sake of simplicity. Notice the usage of `returnURL` hidden input type, which is populated by the AMPByExample server via server-side templating. The server reads this value from a parameter called `return`, automatically added by the AMP library to the sign-in URL.
+
+Find below an example of the value for the `return` parameter added to the request once you click on the login button. You can explore this value by using the Chrome dev console navigating to the Network tab.
+
+<amp-img src="/static/img/return-parameter.png" alt="Return parameter" height="150" width="600"></amp-img>
+
+
+Once the AMPByExample server receives the POST request from the login page and the login and password are correct, it redirects to the `returnURL` we mentioned above appending the parameter `#success=true`. The AMP runtime can now authorize the page and finally allow you to add a comment.
```

Comment:
Minor fixes, rewrite as follows:

Once the AMPByExample server receives the POST request from the login page and the login and password are correct, it redirects the request to the `returnURL` that we mentioned above, and appends the `#success=true` parameter .